lib/service_wms.c

```diff
@@ -728,7 +733,7 @@ void _mapcache_service_wms_parse_request(mapcache_context *ctx, mapcache_service
       goto proxies;
     } else {
       mapcache_tileset *tileset = mapcache_configuration_get_tileset(config,str);
-      if(!tileset) {
+      if(!tileset || tileset->format->type == GC_RAW) {
```

Comment:
not checking tileset->format!=NULL might be the cause of the crash in https://github.com/mapserver/mapserver/issues/5596#issuecomment-454412529_